index.js

```diff
@@ -109,7 +132,8 @@ exports.flattenThread = function (thread) {
     addedIds.add(msg.key)
   }
   function flattenAndReorderReplies (msg) {
-    if (msg.value.content.type == 'post' && isaReplyTo(msg, thread)) {
+    if (thingsThatArePosts.indexOf(msg.value.content.type !== -1) && 
```

Comment:
The problem isnt what that condition *intends* to do, it's just that the conditional is mis-written. It should be:

```js
if (thingsThatArePosts.indexOf(msg.value.content.type) !== -1 && ...
```

The equality check against `-1` occurs outside of the parameters for `indexOf`